### ISSUE TITLE
Changing dynamic during arpeggiation

### ISSUE THREAD

**Issue by Dima12357** [musescore/MuseScore#22377]
### Your idea

I propose to expand the notation options (currently available in Musescore) of dynamic notation to include arpeggiation.
It would be great if the user could place the hairpins vertically right in front of the arpeggio sign, thereby indicating how the loudness should change during the playback of the arpeggiated chord.
![Снимок экрана (228)](https://github.com/musescore/MuseScore/assets/134041121/80634d0c-3d74-4e52-be7a-60c125fa74ea)

### Problem to be solved

This will allow the reader/performer to clearly understand exactly how the arpeggiated chord should be played. And it will be easier for the user to compose such a musical notation, instead of entering a bunch of notes before the beat and indicating the dynamics. I suggest a simpler and more understandable version of the notation.

### Additional context

It should be understood that the change in dynamics during arpeggiation occurs taking into account the fact that the chord is played before the beat (and the last note falls on the beat), which has not yet been fixed in Musescore, as I outlined in issue #22342.

**Comment by mmeyn** [comment]
I’ve never seen such a thing. Are there any existing scores that use that notation? Is it a standard notation for this use case which several composers or engravers have agreed on?

**Comment by mmeyn** [comment]
Hm … just realized that at least Elaine Gould describes that notation.

**Comment by Dima12357** [comment]
> Hm … just realized that at least Elaine Gould describes that notation.

Yes, that's right. Something similar has been described about the diagonal matching of hairpins along the glissando line, but I still can't find where this option is consistent and standardized.

**Comment by DaddyLudwig** [comment]
I've never seen that kind of notation either. The only time I've seen a crescendo or diminuendo with an arpeggiated chord is if the arpeggiation is measured a la Moonlight Sonata Presto agitato, i.e. something like this:
![image](https://github.com/musescore/MuseScore/assets/6530761/b669863b-c551-4209-a22b-f81784c3d1fb)

**Comment by Dima12357** [comment]
> I've never seen that kind of notation either. The only time I've seen a crescendo or diminuendo with an arpeggiated chord is if the arpeggiation is measured a la Moonlight Sonata Presto agitato, i.e. something like this: ![image](https://private-user-images.githubusercontent.com/6530761/322225656-b669863b-c551-4209-a22b-f81784c3d1fb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMwMzY3OTUsIm5iZiI6MTcxMzAzNjQ5NSwicGF0aCI6Ii82NTMwNzYxLzMyMjIyNTY1Ni1iNjY5ODYzYi1jNTUxLTQyMDktYTIyYi1mODE3ODRjM2QxZmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MTNUMTkyODE1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWVjMDliMGY4MzY5OTQ4MWZhOGM5YmRlYmU5MTVlNjkxMjFlNDRjZDI2YTJhYTIxZWRkYzE1YzYyZDlkMGJhNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.XiGAUogFkzW3y3p4pk2ZdSPT50JuD_A40EDKe6X7_Ck)

The example you gave is great for comparing the ways of writing this method. I suggest a way that will save horizontal space. Of course, you can write it down as shown in your picture, but it takes up a lot of space.

**Comment by Dima-S-Jr** [comment]
> Are there any existing scores that use that notation?

This notation can be found in the Japanese composer Toru Takemitsu in his work Les Yeux Clos II.
![Снимок экрана (344)](https://github.com/musescore/MuseScore/assets/134041121/21ae7a15-22cb-4ea8-aa31-3867ffcb9bc7)

There are other examples, but I still can't remember where exactly. If they get caught, I'll let you know.
